### ISSUE TITLE
NTR 'material consumption' & 'material extraction'

### ISSUE THREAD

**Issue by mark-jensen** [SDG-InterfaceOntology/sdgio#84]
@pdez90 requests

>We do not have the term domestic material consumption in the portal. We have 'land consumption' in ENVO but thats about it.
Therefore we need to add the terms 'material consumption', 'material extraction' to the ontology. I'm not sure whether for the purposes of tagging we need to create a specific tag for 'domestic material consumption', as opposed to material consumption.
We use the term domestic material consumption fairly frequently, however.

**Comment by mark-jensen** [comment]
The Eurostat [page](http://ec.europa.eu/eurostat/statistics-explained/index.php/Material_flow_accounts_and_resource_productivity) on Material Flow accounts is helpful

'domestic material consumption' is defined as
> the annual quantity of raw materials extracted from the domestic territory, plus all physical imports minus all physical exports

Which is the quite close to [this](http://www.un.org/esa/sustdev/natlinfo/indicators/methodology_sheets/consumption_production/domestic_material_consumption.pdf) from ESA
> the total amount of materials directly used in the economy (used domestic extraction plus imports), minus the materials that are exported

**Comment by pdez90** [comment]
Hi,

I just checked with Lud. These are indeed definitions that we can use.
Sadly, we were not given the metadata for these datasets and therefore
using this is the best we can do.

We also need to define material consumption per capita and have it in the
ontology


On Thu, Feb 18, 2016 at 1:11 AM, Mark Jensen <notifications@github.com>
wrote:

> The Eurostat page
> <http://ec.europa.eu/eurostat/statistics-explained/index.php/Material_flow_accounts_and_resource_productivity>
> on Material Flow accounts is helpful
>
> 'domestic material consumption' is defined as
>
> the annual quantity of raw materials extracted from the domestic
> territory, plus all physical imports minus all physical exports
>
> Which is the quite close to this
> <http://www.un.org/esa/sustdev/natlinfo/indicators/methodology_sheets/consumption_production/domestic_material_consumption.pdf>
> from ESA
>
> the total amount of materials directly used in the economy (used domestic
> extraction plus imports), minus the materials that are exported
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SDG-InterfaceOntology/sdgio/issues/84#issuecomment-185430446>
> .
>



-- 
Yours Sincerely,
Priyanka deSouza

**Comment by pdez90** [comment]
http://unstats.un.org/sdgs/files/metadata-compilation/Metadata-Goal-12.pdf has this definition:

Domestic Material Consumption (DMC) is a standard material flow accounting
(MFA) indicator and reports the apparent consumption of materials in a national
economy. It is calculated as direct imports (IM) of material plus domestic extraction
(DE) of materials minus direct exports (EX) of materials measured in metric tonnes.
DMC measures the amount of materials that are used in economic processes. It does not
include materials that are mobilized the process of domestic extraction but do not enter
the economic process.
DMC is based on official economic statistics and it requires some modelling to adapt the
source data to the methodological requirements of the MFA. The accounting standard
and accounting methods are set out in the EUROSTAT guidebooks for MFA accounts
in the latest edition of 2013. MFA accounting is also part of the central framework of
the System of integrated Environmental-Economic Accounts (SEEA).

**Comment by pbuttigieg** [comment]
Given that [eurostat says](http://ec.europa.eu/eurostat/statistics-explained/index.php/Material_flow_accounts_and_resource_productivity)
>Natural resources include biomass, metal ores, non-metallic minerals and fossil energy materials. 

I think we should import materials from ENVO and add the SDGIO resource role to them to axiomatise these classes. To address this issue immediately, I think it is best to create `material extraction process` and `material consumption process` in ENVO and then `resource extraction process` and `resource consumption process` in SDGIO (with UNEP preferred synonyms to match the desired labels). This would allow us flexibility later where some regulations would call material X a resource and others don't. The latter can live in ENVO and still be represented.

**Comment by pbuttigieg** [comment]
import and export processes are SDGIO turf, until we have the relevant ontologies running.

**Comment by pbuttigieg** [comment]
@pdez90 are we only concerned with DMC or MC in general?

**Comment by pbuttigieg** [comment]
This paper by [Behrens et al.](http://www.sciencedirect.com/science/article/pii/S0921800907001681) looks like a useful reference here.

**Comment by pdez90** [comment]
Im not sure. We need to tag the dataset Domestic Material Consumption. Im
not sure whether it should be tagged as DMC or MC

I looked at the UN Bis thesaurus to see if either term was present but
couldnt find anything:
http://lib-thesaurus.un.org/LIB/DHLUNBISThesaurus.nsf/$$searche?OpenForm

Therefore I think we have to take a call on what to tag the dataset as. If
you think the meaning is vastly different between the terms then we should
tag it as DMC instead of MC

On Thu, Feb 18, 2016 at 7:24 PM, Pier Luigi Buttigieg <
notifications@github.com> wrote:

> This paper by Behrens et al.
> <http://www.sciencedirect.com/science/article/pii/S0921800907001681>
> looks like a useful reference here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SDG-InterfaceOntology/sdgio/issues/84#issuecomment-185800987>
> .
>



-- 
Yours Sincerely,
Priyanka deSouza

**Comment by pbuttigieg** [comment]
@pdez90 
In this case we can create both, one as a subclass of the other. We just needed to know if the intention was a substitution. I'd recommend using the more specific one (DMC) for a DMC data set.

**Comment by mark-jensen** [comment]
I agree, adding `domestic material consumption` and will hold on adding `material consumption` until we find out.

`domestic material consumption` =
A data item which represents the total amount of materials directly used in a national economy, measured in metric tonnes, and computed via direct imports plus materials extracted minus direct exports.

**Comment by pbuttigieg** [comment]
@mark-jensen 
As discussed elsewhere, ENVO will have a `material extraction process` and SDGIO can have a subclass which references those materials which the SDG process is concerned with. However, note that the requested terms are not processes, but similar to OBI `predicted values` with metric tonnes as units.

Due to the SDG-specificity of material consumption, this should be an SDGIO class, perhaps more accurately labelled `resource consumption` with `material consumption` as the UNEP preferred label.

`resource consumption `= A predicted value which estimates the metric tonnage of resource materials imported into or extracted from an environmental system and used as input for economic processes in that system, minus the tonnage of materials exported from that system.
(a bit long winded, edits welcome)

`domestic resource consumption`= A resource consumption value which pertains to the environmental systems which overlap with a national economic system. 

Classes needed to flesh this out:
- (domestic) import process
- (domestic) export process
- (domestic) resource extraction process (sublcass of ENVO's `material extraction process`)

domestic classes can be either asserted or inferred subclasses.

**Comment by pbuttigieg** [comment]
The labels should actually be `resource consumption estimate` etc. with UNEP preferred labels as requested.

**Comment by mark-jensen** [comment]
@pbuttigieg I added `resource consumption`, changed labels and defs as requested, adding UNEP preferred labels.

@pdez90 Let us know if that works.
